Update PhabricatorKoreanTranslation.php

Katinor opened the pull request:
(^o^)/

Katinor committed:
Update PhabricatorKoreanTranslation.php

revi commented:
> 다른 사람을 방해하고 있습니다

실제 용례에서는 방해한다기보다 다른 사람의 작업을 가로막는다는 의미에서 사용됩니다.

LiteHell commented:
> ```
>    'Added a subscriber: %2$s.',
>    '구독자 추가: %2$s.',
> ```

_구독자를 추가했습니다: %2$s_ 혹은 _구독자 추가됨: %2$s_ 둘 중 하나와 같은 형식으로 번역하는 게 더 자연스러울 수도 있습니다. 어디에 어떨때 표시(출력)되는가에 따라 다르지만요.

LiteHell commented:
> ```
>    '%s Open Pull Request',
>    '%s Open Pull Requests',
>    '열린 분기점 합병 요구 %s개',
>    '열린 분기점 합병 요구 %s개',
> ```

번역이지만 억지로 한국어로 번역할 필요는 없습니다. 그냥 **풀 리퀘스트**라고 번역하는 게 더 어울릴 같네요.

> ```
>    '열린 풀 리퀘스트 %s개'
> ```

이런 식으로요.

LiteHell commented:
- 차단된 작업 (blocked task)
- 차단중인 작업 (blocking task)

제가 보기엔 오역같습니다. 제가 틀릴 수도 있지만요.

LiteHell commented:
> id:%s인 데몬이 존재하지 않습니다

좋은 번역이지만 다음과 같이 수정하면 더 좋아질 거 같아요 ;)

> id가 %s인 데몬이 존재하지 않습니다.

LiteHell commented:
> 성공적으로 다른점 %d개를 만들었습니다.

다른 번역이 가능하다면 다른 번역으로 하는 게 더 좋을 것 같습니다.

Katinor committed:
Update PhabricatorKoreanTranslation.php

1. id: -> id가 ~
2. diff의 번역으로 "다른점" 대신 "차이"를 사용했습니다.
3. Line 69의 번역을 바꾸었습니다.
4. blocking/blocked에 대해 새로운 번역을 사용했습니다. Line 209부터입니다.
5. pull request의 번역으로 "분기점 합병 요구" 대신 "풀 리퀘스트"를 사용했습니다.
6. Line 1305부터 적용된 딱딱한 번역에 "됨"을 붙였습니다.
7. 앞뒤로 어투가 다른 부분을 통일하였습니다: ~를 위해 -> "~"의 사유로

Katinor committed:
Update PhabricatorKoreanTranslation.php

Katinor committed:
Update PhabricatorKoreanTranslation.php

Application, Create Task 번역완료

Katinor committed:
Update PhabricatorKoreanTranslation.php

Katinor committed:
Update PhabricatorKoreanTranslation.php

for을 ~라는 이유로, ~위해로 해석한사람 누구에요 ㅠㅠㅠㅠㅠㅠㅠㅠㅠㅠ

Katinor committed:
Update PhabricatorKoreanTranslation.php

현재 버젼에서 사용중인 매니페스트 기능 번역

revi commented:
@homu retry

revi commented:
@homu r+

일단 병합하고 계속 수정하는 걸로 하죠. (양이 너무 많아서...)

homu commented:
:pushpin: Commit e8928e1 has been approved by `revi`

<!-- @homu r=revi e8928e1f10f805c05e8a4c9158d9707d2382a190 -->

homu commented:
:hourglass: Testing commit e8928e1 with merge bb939eb...

homu commented:
:broken_heart: Test failed - [status](https://travis-ci.org/librewiki/phabricator-korean-i18n/builds/99600653)

Katinor committed:
Update PhabricatorKoreanTranslation.php

revi commented:
@homu r+

homu commented:
:pushpin: Commit f0613fb has been approved by `revi`

<!-- @homu r=revi f0613fb65bdca8011b4e2588bc365b51abcb7644 -->

homu commented:
:zap: Test exempted - [status](https://travis-ci.org/librewiki/phabricator-korean-i18n/builds/99602439)
